x/concentrated-liquidity/keeper.go

```diff
@@ -118,13 +118,10 @@ func (k *Keeper) SetListeners(listeners types.ConcentratedLiquidityListeners) *K
 	return k
 }
 
-// ValidatePermissionlessPoolCreationEnabled returns nil if permissionless pool creation in the module is enabled.
+// IsPermissionlessPoolCreationEnabled returns nil if permissionless pool creation in the module is enabled.
 // Otherwise, returns an error.
```

Comment:
```suggestion
// IsPermissionlessPoolCreationEnabled returns true if permissionless pool creation in the module is enabled.
// Otherwise, returns false
```